src/kotlin/kage/format/AgeKey.kt

```diff
@@ -55,6 +55,9 @@ public class AgeKey(
 
       if (privateKey.isEmpty())
         throw InvalidAgeKeyException("Cannot find private key in age key file")
+
+      // TODO: The private key needs to be decoded from Bech32
+      // TODO: The public key needs to be decoded from Bech32
       return AgeKey(created, publicKey.encodeToByteArray(), privateKey.encodeToByteArray())
```

Comment:
I did try to parse this as Bech32 using https://github.com/komputing/KBech32 and even https://raw.githubusercontent.com/bitcoinj/bitcoinj/master/core/src/main/java/org/bitcoinj/base/Bech32.java But this was giving me wrong values, it was not decoding it properly. I don't know if age.go implements a different version of bech32, or what is going on here, but we cannot decode this as base64.

Reply:
I'll take a look at the spec.